src/arch/pmp.rs

```diff
@@ -63,13 +94,19 @@ pub const fn build_napot(start: usize, size: usize) -> Option<usize> {
     Some((start >> 2) | ((size - 1) >> 3))
 }
 
+pub const fn build_napot_pair(pair: (usize, usize)) -> Option<usize> {
+    build_napot(pair.0, pair.1)
+}
+
 // ——————————————————————————————— PMP Group ———————————————————————————————— //
 
 pub struct PmpGroup {
     pmpaddr: [usize; 64],
     pmpcfg: [usize; 8],
+    modified: bool,
```

Comment:
You should also set it to `true` in the `set` function used by miralis, not only in `set_from_policy`.

Reply:
This is already the case :-)